miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -163,6 +164,17 @@ open class MiniAppMessageBridge {
         throw CustomPermissionsNotImplementedException()
     }
 
+    /**
+     * handle Universal Bridge that represented as a json  from MiniApp.
+     **/
+    open fun sendJsonToHostApp(
+        jsonStr: String,
+        onSuccess: (jsonStr: String) -> Unit,
+        onError: (message: String) -> Unit
+    ) {
+        throw JsonInfoNotImplementedException()
```

Comment:
@shohiebsense you can use this common implementation for consistancy
```
        throw MiniAppSdkException(ErrorBridgeMessage.NO_IMPL)
```
we generally throw this exception if any funtion is not overridden. I dont think there is a need to add a new one for this.